### ISSUE TITLE
Live Preview with static caching enabled

### ISSUE THREAD

**Issue by FrittenKeeZ** [statamic/v2-hub#2153]
Seemingly this should work with `sample.htaccess` but `sample.nginx.conf` doesn't have a check for request method being `GET` only.

This should resolve the issue for nginx, so live preview once again can be enabled for static caching.
```
# Dynamically detect request of static cached pages.
set $dynamic_location @default;
if ($request_method = GET) {
    set $dynamic_location @static;
}
location / {
    try_files $uri $dynamic_location;
}
location @static {
    try_files /static${uri}_${query_string}.html @default;
}
location @default {
    try_files $uri $uri/ /index.php?$query_string;
}
```

**Comment by FrittenKeeZ** [comment]
To test it out locally and force live preview back on with static caching enabled, set `'staticCachingEnabled': false,` in `statamic/resources/views/partials/head.blade.php` and enable static caching in `site/settings/caching.yaml`:
```yaml
static_caching_enabled: true
static_caching_type: file
```

**Comment by philipboomy** [comment]
I have tested this and I can confirm it works for me.

**Comment by FrittenKeeZ** [comment]
@jackmcdade @jasonvarga have you had any time to test this?

**Comment by bgarrant** [comment]
It would be great to get Live Preview and caching working together.  It makes flat file benefits minimal without the caching on.  Clients love the Live Preview so it should be usable all the time.

**Comment by bgarrant** [comment]
@jackmcdade  @jasonvarga Is there any way to get this working in V2?  If not in V2 how about V3?

**Comment by jasonvarga** [comment]
Released in 2.10.7
